app-crypt/acme-sh/files/99acme-sh

```diff
@@ -0,0 +1,2 @@
+export LE_WORKING_DIR="/etc/acme-sh/"
```

Comment:
We don't `export` stuff here, it's implicitly exported.

Reply:
Removed export.